### ISSUE TITLE
using quantum-journal for a submission?

### ISSUE THREAD

**Issue by araujoms** [cgogolin/quantum-journal#29]
Currently quantum-journal makes the title of the paper a link to a search within quantum-journal.org. This makes it awkward to use the style for a submission, as at the time of submission this link is necessarily dead. I suggest to activate the link only after a paper is accepted, and with a DOI, not a search, so that it becomes more reliable.

This would also make it easier to use quantum-journal as a neutral style for typesetting papers, as REVTeX is often used.

**Comment by cgogolin** [comment]
Actually this is kind of intentional. We don't want people to use `quantumarticle.cls` to typeset there generic papers. There are enough good generic templates out there to do that. The quantumarticle class with its specific look and feel is meant to help with establishing a brand image for Quantum.

The problem with activating the link only after submission is that it is something the authors would have to do (because they upload the final version to the arXiv) and it will be a nightmare to make sure this is always done and authors will not be very happy to hear that they have to replace on the arXiv again only because they forgot to activate the link.

Sorry, but I don't think we will fix this.

**Comment by araujoms** [comment]
I understand your point. Can I then ask you to gently discourage authors from submitting papers written with quantum-journal? I think this would then fix the problem of the dead link, of the use for generic papers, and of the extremely embarrassing situation somebody would be in if the person submitted a paper written in the quantum-journal style and had the paper rejected by Quantum afterwards.

**Comment by Strilanc** [comment]
+1 to fixing this

I can see the value of having a link in the paper back to some meta page about the paper. But why put the link *under the title*? And why is the destination *a search page*? This is a very unexpected result of clicking on the title. It's bad UX.

I would recommend removing the link even in the published articles. Or at least move the link to a subheader that says something like "Published in X". Or split the style into draft/published parts.

**Comment by cgogolin** [comment]
The problem is that at the time the pdf is produced (by the authors locally on their machine or by the arXiv) no permanent identifier of the paper can be automatically determined. The DOI has not yet been assigned and there is no way to access the arXiv number from within LaTeX. Obviously we also have no means of retroactively changing the version uploaded to the arXiv once such identifiers have been assigned and adding such an identifier would hence require manual intervention from the authors and in most cases require the hassle of uploading an additional iteration to the arXiv.

The only (almost) unique way to automatically identify a manuscript at compile time is via its title. Therefore, the only way to provide a mechanism for our readers to check whether a manuscript has been published by Quantum that we could think of is via a search for that paper's title on quantum-journal.org.

It is very natural that a search for the paper's title can be triggered by clicking the title (and any decent PDF reader will show you the url the title links to when hovering over it with the mouse), so I don't think this is "unexpected" or "bad UX". On the contrary, moving the link to a subheader that says "Published in Quantum" would be a very very bad idea, because it would not be true for papers rejected or under review and it would be much more problematic to have a paper with such a wrong and misleading claim on the arXiv in case of rejection.

If you have a better suggestion for how to provide such a mechanism without requiring manual intervention from the authors, please let us know.

**Comment by cgogolin** [comment]
On top of the above:
- Nobody forces you to use quantumarticle.cls for the initial submission to Quantum. We only highly recommend it for the final published version. If you find it embarrassing, to have a preprint on the arXiv that uses quantumarticle.cls but then was not accepted, just don't use quantumarticle.cls. Besides, even without the link it wouldn't get any less obvious that the paper was intended for Quantum as the style would still be clearly visible.
- The link back to our page for verification of publication is especially important on manuscripts that were not published by Quantum. It is these cases where we want there to be an easy way for readers to check that that work did not meet our quality standards. For good papers that were accepted the feature is there just for convenience.

**Comment by dbranford** [comment]
I'd favour introducing a draft/preprint/unpublished key which would disable some of these features which only make sense for an accepted manuscript and remove (some of) the default Quantum house style.  This would enable authors to produce neutral-looking articles which can easily be converted to the current `quantumarticle` style (through a single key change) with limited chance of failure (given the same backend is used) upon acceptance.

Submitting an article with one class when the final article should be produced (by the authors) with a different class makes little sense to me, there's too much room for incompatibilities between classes (although I acknowledge a lot of thought has gone into ensuring compatibility, particularly with `revtex`, ) to be worth risking.  Similarly, authors wishing to use `biblatex` as permitted by `quantumarticle` would find themselves unable to use a class like `revtex` and potentially other freedoms offered by `quantumarticle` may not be easily enacted with other classes, yet starting from a basic class like `article` is likely to be deemed an excessive amount of work pushing authors again towards using `quantumarticle` for submissions.

**Comment by cgogolin** [comment]
That sounds like a useful feature. It would have to make the paper sufficiently "ugly" so that authors don't forget to turn this off for the final version.
However, we shouldn't discuss this here, but rather open a new issue, please.
Do you want to do that (to take credit for it) or shall?
